src/ast/data_type.rs

```diff
@@ -337,6 +341,7 @@ fn format_datetime_precision_and_tz(
 /// guarantee compatibility with the input query we must maintain its exact information.
 #[derive(Debug, Copy, Clone, PartialEq, PartialOrd, Eq, Ord, Hash)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[cfg_attr(feature = "visitor", derive(Visit))]
```

Comment:
I am a big fan of this method as a way to minimize maintenance burden on new contributions

I picked a random struct and removed the `#[cfg_attr(feature = "visitor", derive(Visit))]` to see what would happen if it was forgotten

I think the error is fairly clear 👍 

```
   Compiling sqlparser v0.28.0 (/Users/alamb/Software/sqlparser-rs)
error[E0277]: the trait bound `ast::Password: visitor::Visit` is not satisfied
    --> src/ast/mod.rs:1262:9
     |
1262 |         password: Option<Password>,
     |         ^^^^^^^^ the trait `visitor::Visit` is not implemented for `ast::Password`
     |
     = help: the following other types implement trait `visitor::Visit`:
               Box<T>
               CommentObject
               CreateTableBuilder
               Keyword
               Option<T>
               String
               Vec<T>
               ast::Action
             and 120 others
note: required for `Option<ast::Password>` to implement `visitor::Visit`
    --> src/ast/visitor.rs:21:16
     |
21   | impl<T: Visit> Visit for Option<T> {
     |                ^^^^^     ^^^^^^^^^
```

Reply:
It's also making it easier to keep up with the merge conflicts :laughing: